react/components/ModalDialog/index.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types'
 import Modal from '../Modal'
 import Button from '../Button'
 
-const NOOP = () => {}
+const NOOP = () => ({})
```

Comment:
You can ignore the rule here to prevent creating an object without reason!

Reply:
Done! Thanks